docs/tutorials/hello.rst

```diff
@@ -64,3 +64,9 @@ change the scroll method's argument.
     in a way that MicroPython can't understand. Check you're not missing any
     special characters like ``"`` or ``:``. It's like putting. a full stop in
     the middle of a sentence. It's hard to understand exactly what you mean.
+
+    Your microbit may stop responding: you cannot flash new code to it or
```

Comment:
Did you experience such behaviour?  It should be that you can always flash new code because the chip that does the flashing is independent from the nRF chip that runs the code.